src/style.css

```
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  height: 15px;
+  display: flex;
+}
+
+.stars__item {
+  width: 15px;
+  height: 15px;
+  background: url("./images/star.svg") center no-repeat;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__item:nth-child(-n+1),
```

```suggestion
.stars--1 .stars__item:first-child,
```
it's easier to understand